embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/metadata/EmbraceMetadataService.kt

```diff
@@ -107,102 +106,111 @@
+                    val displayMetrics = DisplayMetrics()
+                    display?.getMetrics(displayMetrics)
+                    String.format(Locale.US, "%dx%d", displayMetrics.widthPixels, displayMetrics.heightPixels)
+                } catch (ex: Exception) {
+                    null
+                }
                 preferencesService.screenResolution = screenResolution
-                logDeveloper("EmbraceMetadataService", "Screen resolution computed and stored")
             }
         }
     }
 
     private fun asyncRetrieveIsJailbroken() {
-        logDeveloper("EmbraceMetadataService", "Async retrieve Jailbroken")
-
         // if the isJailbroken property exists in memory, don't try to retrieve it
         if (isJailbroken != null) {
-            logDeveloper("EmbraceMetadataService", "Jailbroken already exists")
             return
         }
         metadataBackgroundWorker.submit {
-            logDeveloper("EmbraceMetadataService", "Async retrieve jailbroken")
             val storedIsJailbroken = preferencesService.jailbroken
             // load value from shared preferences
             if (storedIsJailbroken != null) {
-                logDeveloper("EmbraceMetadataService", "Jailbroken is present, loading from store")
                 isJailbroken = storedIsJailbroken
             } else {
                 isJailbroken = MetadataUtils.isJailbroken()
                 preferencesService.jailbroken = isJailbroken
-                logDeveloper("EmbraceMetadataService", "Jailbroken processed and stored")
             }
-            logDeveloper("EmbraceMetadataService", "Jailbroken: $isJailbroken")
         }
     }
 
-    fun asyncRetrieveDiskUsage(isAndroid26OrAbove: Boolean) {
+    private fun asyncRetrieveDiskUsage(isAndroid26OrAbove: Boolean) {
         metadataBackgroundWorker.submit {
-            logDeveloper("EmbraceMetadataService", "Async retrieve disk usage")
             val free = MetadataUtils.getInternalStorageFreeCapacity(statFs.value)
             if (isAndroid26OrAbove && configService.autoDataCaptureBehavior.isDiskUsageReportingEnabled()) {
-                val deviceDiskAppUsage = MetadataUtils.getDeviceDiskAppUsage(
+                val deviceDiskAppUsage = getDeviceDiskAppUsage(
                     storageStatsManager,
                     packageManager,
                     packageName
                 )
                 if (deviceDiskAppUsage != null) {
-                    logDeveloper("EmbraceMetadataService", "Disk usage is present")
                     diskUsage = DiskUsage(deviceDiskAppUsage, free)
                 }
             }
             if (diskUsage == null) {
                 diskUsage = DiskUsage(null, free)
             }
-            logDeveloper("EmbraceMetadataService", "Device disk free: $free")
         }
     }
 
+    @TargetApi(Build.VERSION_CODES.O)
+    @SuppressWarnings("deprecation")
+    fun getDeviceDiskAppUsage(
```

Comment:
Pull this out of the util class because this is the only use case, and putting it here allows me to use the instance's logger. This pattern is repeated a couple times later.